scripts/testing/xbox-ssl.conf

```diff
@@ -47,12 +47,8 @@
     # https://tools.ietf.org/html/rfc5785
     #
 
-    <Location /.well-known/dxos>
-      SetEnv proxy-sendcl
-      ProxyPass "http://localhost:9000/.well-known/dxos.json"
-      ProxyPassReverse "http://localhost:9000/.well-known/dxos.json"
-      Allow from all
-    </Location>
+    RewriteRule ^/.well-known/dxos$ http://localhost:9000/.well-known/dxos.json [P,L]
```

Comment:
It would be helpful to have a short comment introducing the concept of using RewriteRule to implement complex reverse proxy functionality. For example I'm fairly familiar with Apache but I had no clue that RewriteRule can perform proxing. Probably reference the doc for RewriteRule as proxy?